src/lib.rs

```diff
@@ -29,6 +29,7 @@
 \* -------------------------------------------------------------------------- */
 
 #![warn(clippy::unwrap_used)]
+#![allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
This is a crate wide allow; maybe that's your intention, but figured I'd point it out in case.